MapboxNavigation/NavigationMapView.swift

```diff
@@ -440,13 +435,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     /**
      Adds or updates both the route line and the route line casing
      */
-    @objc public func showRoutes(_ routes: [Route], legIndex: Int = 0) {
+    public func showRoutes(_ routes: [Route], legIndex: Int = 0) {
```

Comment:
By the way, now that we only support Swift, we can use more concise method names like:

```swift
public func show(_ routes: [Route], legIndex: Int = 0) {
```

without worrying about conflicts with similarly named methods that take different arguments.